I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 2, in script
    define m = Chaacter('Me', color = "#c8c8ff")
  File "game/script.rpy", line 2, in <module>
    define m = Chaacter('Me', color = "#c8c8ff")
NameError: name 'Chaacter' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 2, in script
    define m = Chaacter('Me', color = "#c8c8ff")
  File "C:\Users\Shiver\Documents\renpy-8.2.1-sdk\renpy\ast.py", line 2219, in execute
    self.set()
  File "C:\Users\Shiver\Documents\renpy-8.2.1-sdk\renpy\ast.py", line 2233, in set
    value = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "C:\Users\Shiver\Documents\renpy-8.2.1-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 2, in <module>
    define m = Chaacter('Me', color = "#c8c8ff")
NameError: name 'Chaacter' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.2.1.24030407
EdensCrucible 1.0
Fri May 17 16:06:07 2024
